DigitalLearningSolutions.Web/Helpers/ExternalApis/MapsApiHelper.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers.ExternalApis
+{
+    using System.Net.Http;
+    using System.Threading.Tasks;
+    using DigitalLearningSolutions.Data.Models.External.Maps;
+    using Newtonsoft.Json;
+
+    public interface IMapsApiHelper
+    {
+        Task<MapsResponse> GetPostcodeCoordinates(string postcode);
+    }
+
+    public class MapsApiHelper : IMapsApiHelper
+    {
+        private readonly HttpClient client;
+        private readonly string mapsApiBaseUrl;
+
+        public MapsApiHelper(HttpClient httpClient)
+        {
+            var mapsApiKey = ConfigHelper.GetAppConfig()["MapsAPIKey"];
+            mapsApiBaseUrl =
+                $"https://maps.googleapis.com/maps/api/geocode/json?key={mapsApiKey}&components=country:GB|postal_code:";
+            httpClient.DefaultRequestHeaders.Accept.Clear();
+            client = httpClient;
+        }
+
+        public async Task<MapsResponse> GetPostcodeCoordinates(string postcode)

```

Comment:
Should we either call it something less specific than "PostcodeCoordinates" or maybe add a new public method that calls this and does the mapping to coordinates?